### ISSUE TITLE
Export tokens that were added via QR-code

### ISSUE THREAD

**Issue by hex-m** [privacyidea/pi-authenticator#421]
### Top-level intent

I want to export my TOTP tokens.

### Steps to reproduce

1. Add token by scanning a QR-code
2. choose function "Export non-privacyIDEA tokens" -> as file
3. get message: "No token available for export"

### Description

If I add the same token manually (instead of scanning a QR-code) I can export it. That seems counterintuitive from a users point of view.

According to [that string](https://github.com/privacyidea/pi-authenticator/blob/master/lib/l10n/app_en.arb#L692) it is known behaviour.

> Currently only manually added and imported tokens are exportable.

But I think it should be considered as a bug.

### Configuration

Android Version 4.4.0+404009

**Comment by frankmer** [comment]
Yes, this is a known and expected behavior. And we know it is not very user friendly.
The first rule of the token managed by privacyIDEA is that it should only exist on one device. So we do not want that a user can get the secret of the token. In older versions of privacyIDEA the QR-Code had no special marker to identify it as a privacyIDEA token. The problem now is that we have to distinguish between tokens managed by privacyIDEA and private tokens if we want to export private tokens. So how can we do that? The short answer is that we cannot currently guarantee that any OTPAuth token is not a privacyIDEA token.

We know that this system of exporting tokens should not stay forever.
We will discuss if, when and how we can implement a system to export private tokens without risking to export pi tokens.
